components/UI/Select/hooks/useSelectOptions.tsx

```diff
@@ -0,0 +1,17 @@
+import { UseSelectOptionsParams } from '../type';
+
+export const useSelectOptions = <Option,>({
+  options,
+  getLabel,
+}: UseSelectOptionsParams<Option>) => {
+  return (
+    <>
+      {options.map((item: Option, index) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <option key={index} value={index}>
```

Comment:
It seems preferable to set a unique value instead of 'index' without turning off the lint.